openpype/hosts/blender/plugins/publish/extract_thumbnail.py

```diff
@@ -147,10 +148,11 @@ def _fix_output_path(self, filepath):
             return None
 
         if not os.path.exists(filepath):
-            files = glob.glob(f"{filepath}.*.jpg")
+            filepath, ext = os.path.splitext(filepath)
```

Comment:
Use `pathlib`

Reply:
Considering `Path` is used all along for `filepath`, you don't need this line and use `filepath.stem` and `filepath.suffix` instead.

Reply:
Ok `Path `all along look good with `Path.glob`